src/RouterSettings/Ports.js

```diff
@@ -62,9 +81,11 @@ const Ports = () => {
     setError(null);
     setConfirming(false);
     let unexpectedError = false;
+    let count = 0; // to keep track of which set we are on in order to send error info if one occurs
```

Comment:
can you expand on this comment? I'm having some trouble following the logic.

Reply:
This... was left over from when I had been trying to send this command in a loop and decided that wasn't the best way, oops. It's been changed to do what I had actually intended now